tools/restatectl/src/commands/sql.rs

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2023 - 2025 Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::{
+    pin::Pin,
+    task::{Context, Poll},
+    time::Instant,
+};
+
+use anyhow::Context as AnyhowContext;
+use arrow::{
+    array::RecordBatch,
+    buffer::Buffer,
+    error::ArrowError,
+    util::display::{ArrayFormatter, FormatOptions},
+};
+use arrow_ipc::reader::StreamDecoder;
+use cling::prelude::*;
+use futures::{Stream, StreamExt, ready};
+use restate_admin::cluster_controller::protobuf::{
+    QueryRequest, QueryResponse, cluster_ctrl_svc_client::ClusterCtrlSvcClient,
+};
+use restate_cli_util::{
+    _comfy_table::{Cell, Table},
+    c_eprintln, c_println,
+    ui::{
+        console::{Styled, StyledTable},
+        stylesheet::Style,
+    },
+};
+use tonic::{Status, Streaming, codec::CompressionEncoding};
+
+use crate::connection::ConnectionInfo;
+
+#[derive(Run, Parser, Collect, Clone, Debug)]
+#[cling(run = "query")]
+pub struct SqlOpts {
+    query: String,
+}
+
+async fn query(connection: &ConnectionInfo, args: &SqlOpts) -> anyhow::Result<()> {
+    //TODO: clean up
```

Comment:
is this meant to be done in this PR?

Reply:
Ah no it was a placeholder for something that is not their anymore. Sorry about that. Will remove